nautobot_ssot/integrations/infoblox/diffsync/adapters/infoblox.py

```diff
@@ -45,13 +45,56 @@ def __init__(self, *args, job=None, sync=None, conn=None, **kwargs):
             )
             raise PluginImproperlyConfigured
 
+    # credit to @Eric-Jckson for the conainer code addition: https://github.com/nautobot/nautobot-plugin-ssot-infoblox/pull/143
```

Comment:
Can we do `Co-Authored-By` instead? I would prefer not to have this comment here because it might quickly lose meaning compared to the co-authorship

Reply:
Done as part of moving functions to client utils.